ext/bcmath/libbcmath/src/num2str.c

```diff
@@ -56,23 +56,23 @@ zend_string *bc_num2str_ex(bc_num num, size_t scale)
 
 	/* Load the whole number. */
 	nptr = num->n_value;
-	for (index=num->n_len; index>0; index--) {
+	for (index = num->n_len; index > 0; index--) {
```

Comment:
I don't remember if ``n_len`` and `n_scale` are `size_t` or not, if yes the `index` var is causing an implicit sign change

Reply:
Changed the type to `size_t`.